src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -269,11 +328,115 @@ const handleOpenAuth = (mediator, artist) => {
   })
 }
 
+const RoundedFlex = styled(Flex)`
+  border-radius: 100px;
+`
+
+const renderAuctionHighlight = artist => {
+  if (
+    !artist.auctionResultsConnection ||
+    artist.auctionResultsConnection.edges.length < 1
+  ) {
+    return null
+  }
+  const topAuctionResult =
+    artist.auctionResultsConnection.edges[0].node.price_realized.display
+
+  return (
+    <RoundedFlex
+      background={color("black5")}
+      width="auto"
+      py={5}
+      px={10}
+      mr={5}
+    >
+      <AuctionIcon pr={5} />
+      <Sans size="2">{topAuctionResult} Auction Record</Sans>
+    </RoundedFlex>
+  )
+}
+
+const CATEGORIES = {
+  "blue-chip": "Blue chip",
+  "top-established": "Established",
+  "top-emerging": "Emerging",
+}
+
+const ICON_MAPPING = {
+  BLUE_CHIP: BlueChipIcon,
+  TOP_ESTABLISHED: TopEstablishedIcon,
+  TOP_EMERGING: TopEmergingIcon,
+}
+
+const renderRepresentationStatus = artist => {
+  const { highlights } = artist
+  const { partnersConnection } = highlights
+  if (
+    partnersConnection &&
+    partnersConnection.edges &&
+    partnersConnection.edges.length > 0
+  ) {
+    const highCategory = highestCategory(partnersConnection.edges)
+    const type = highCategory.toUpperCase().replace("-", "_")
+
+    return (
+      <RoundedFlex background={color("black5")} width="auto" py={5} px={10}>
```

Comment:
It seems worth refactoring out this component that shows the icon+label (it may be something that eventually shows up in palette).

It could accept the output of your `ICON_MAPPING` const above and a label and render accordingly.